test/integration/targets/inventory_kubevirt_conformance/constraints.txt

```diff
@@ -1,3 +1,4 @@
 setuptools < 45 ; python_version <= '2.7' # setuptools 45 and later require python 3.5 or later
 ruamel.yaml < 0.17 ; python_version <= '2.7'  # ruamel.yaml 0.17 and later are not tested with Python 2.7
 ruamel.yaml.clib <= 0.2.2 ; python_version < '3.5'  # ruamel.yaml.clib 0.2.3 and later requires Python 3.5 or later
+kubernetes < 25.3.0
```

Comment:
Were you able to determine what change in that version causes the test failure? If so, it would be nice to include a brief comment explaining that. If not, just a statement that newer versions don't pass the test would suffice. I wouldn't spend any time figuring out out the "why" unless you already discovered that.

Reply:
Nothing obvious in the changelog. I thought looking in the source code would not be worth the effort.